include/aspect/material_model/rheology/diffusion_creep.h

```diff
@@ -159,7 +198,7 @@ namespace aspect
           /**
            * Diffusion creep grain size d.
```

Comment:
Please extend this to explain that  this is read from the input file, and only used by the functions that do not take the grain size as additional argument.